core/src/main/java/com/linecorp/armeria/common/stream/AbstractStreamMessageDuplicator.java

```diff
@@ -67,26 +70,60 @@
 public abstract class AbstractStreamMessageDuplicator<T, U extends StreamMessage<T>>
         implements SafeCloseable {
 
+    private static final long DEFAULT_MAX_SIGNAL_LENGTH = Flags.defaultMaxResponseLength();
```

Comment:
Well, that is actually why I created maxRequestResponseLength  -_-; . So, do I need to another variable for this? like I told you, I'm afraid of having too many variables in Flags..